SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -240,7 +245,7 @@ final class SwedbankPaySDKViewModel: NSObject {
                 urlRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
                 urlRequest.httpBody = data
                 
-                self?.sessionManager.request(urlRequest).responseJSON(completionHandler: { [weak self] response in
+                self?.sessionManager.request(urlRequest).responseDecodable(of: OperationsList.self, completionHandler: { [weak self] response in
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 144 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)